src/views/Stack/Card.tsx

```diff
@@ -593,9 +608,13 @@ export default class Card extends React.Component<Props> {
                 cond(
                   this.distance,
                   divide(
-                    multiply(
-                      this.gestureUntraversed,
-                      I18nManager.isRTL ? MINUS_ONE_NODE : TRUE_NODE
+                    cond(
+                      eq(this.direction, DIRECTION_HORIZONTAL),
+                      multiply(
+                        this.gestureUntraversed,
+                        I18nManager.isRTL ? MINUS_ONE_NODE : TRUE_NODE
+                      ),
+                      this.gestureUntraversed
```

Comment:
It was another bug here. In RTL we were multiplying a gesture in vertical mode as well.